runtime-modules/proposals/codex/src/proposal_types/mod.rs

```diff
@@ -47,14 +46,8 @@ pub enum ProposalDetails<MintedBalance, CurrencyBalance, BlockNumber, AccountId,
     /// Balance for the `set content working group mint capacity` proposal
     SetContentWorkingGroupMintCapacity(MintedBalance),
 
-    /// AccountId for the `evict storage provider` proposal
-    EvictStorageProvider(AccountId),
-
     /// Validator count for the `set validator count` proposal
     SetValidatorCount(u32),
-
-    /// Role parameters for the `set storage role parameters` proposal
-    SetStorageRoleParameters(RoleParameters<CurrencyBalance, BlockNumber>),
```

Comment:
`_SetStorageRoleParameters(RoleParameters<CurrencyBalance, BlockNumber>)`
This means we also need to preserve the `struct RoleParameters` type.

I think we can remove the encapsulated value if we can ensure we don't try to read in the runtime any proposal detail of this variant.

Reply:
Fixed.